### ISSUE TITLE
Add a crosseller

### ISSUE THREAD

**Issue by israelga** [nexopos/v3-issues#36]
**Is your requested function related to a problem? Please describe.**
Do not

**Describe the solution you want. Would**

When you are going to make a payment for an X product, why not add a pop up that offers other products and increase sales?

Normally a worker like a waiter, a store clerk forget to offer other products and increase sales. Why pop up can not serve as a reminder for the worker and help that business to increase sales?

I would like the technology to work in our favor to make the business more profitable.

We can see the example constantly in Mcdonald, when we order a Bigmac menu and the waitress offers you a larger drink or an extra of chips.

**Describe the alternatives you have considered**
One way that should be done is with a small configuration of associated products to offer.

For example: If I sell a pack of anti-aging creams I associate this product with a facial cleansing product.

**Additional context**

**Comment by Blair2004** [comment]
This feature seems more usefull if we have a website where customers can order. This can brings unecessary complexity to the application.

**Comment by israelga** [comment]
Yes, but the commercial actions of the workers are left to do because they do not remember, however if the Nexopos every time you go to collect it reminds you, maybe it would be an excellent option to increase sales.

**Comment by nexopos** [comment]
Hi, It looks like your issue does'nt actually match the required format.
                        Please note that you can choose between the following format : 
                            - [Issue](https://github.com/nexopos/v3-issues/issues/new?template=bug_report.md).
                            - [Feature Submition](https://github.com/nexopos/v3-issues/issues/new?template=feature_request.md).
                        You might consider updating your issue and use the right version format. 
                        Regards.

**Comment by nexopos** [comment]
Hi, It looks like your issue does'nt actually match the required format.
                        Please note that you can choose between the following format : 
                            - [Issue](https://github.com/nexopos/v3-issues/issues/new?template=bug_report.md).
                            - [Feature Submition](https://github.com/nexopos/v3-issues/issues/new?template=feature_request.md).
                        You might consider updating your issue and use the right version format. 
                        Regards.

**Comment by nexopos** [comment]
Hi, It looks like your issue does'nt actually match the required format.
                        Please note that you can choose between the following format : 
                            - [Issue](https://github.com/nexopos/v3-issues/issues/new?template=bug_report.md).
                            - [Feature Submition](https://github.com/nexopos/v3-issues/issues/new?template=feature_request.md).
                        You might consider updating your issue and use the right version format. 
                        Regards.

**Comment by nexopos** [comment]
Hi, It looks like your issue does'nt actually match the required format.
                        Please note that you can choose between the following format : 
                            - [Issue](https://github.com/nexopos/v3-issues/issues/new?template=bug_report.md).
                            - [Feature Submition](https://github.com/nexopos/v3-issues/issues/new?template=feature_request.md).
                        You might consider updating your issue and use the right version format. 
                        Regards.

**Comment by nexopos** [comment]
Hi, It looks like your issue does'nt actually match the required format.
                        Please note that you can choose between the following format : 
                            - [Issue](https://github.com/nexopos/v3-issues/issues/new?template=bug_report.md).
                            - [Feature Submition](https://github.com/nexopos/v3-issues/issues/new?template=feature_request.md).
                        You might consider updating your issue and use the right version format. 
                        Regards.

**Comment by nexopos** [comment]
Hi, It looks like your issue does'nt actually match the required format.
                        Please note that you can choose between the following format : 
                            - [Issue](https://github.com/nexopos/v3-issues/issues/new?template=bug_report.md).
                            - [Feature Submition](https://github.com/nexopos/v3-issues/issues/new?template=feature_request.md).
                        You might consider updating your issue and use the right version format. 
                        Regards.

**Comment by nexopos** [comment]
Hi, It looks like your issue does'nt actually match the required format.
                        Please note that you can choose between the following format : 
                            - [Issue](https://github.com/nexopos/v3-issues/issues/new?template=bug_report.md).
                            - [Feature Submition](https://github.com/nexopos/v3-issues/issues/new?template=feature_request.md).
                        You might consider updating your issue and use the right version format. 
                        Regards.

**Comment by nexopos** [comment]
Hi, It looks like your issue does'nt actually match the required format.
                        Please note that you can choose between the following format : 
                            - [Issue](https://github.com/nexopos/v3-issues/issues/new?template=bug_report.md).
                            - [Feature Submition](https://github.com/nexopos/v3-issues/issues/new?template=feature_request.md).
                        You might consider updating your issue and use the right version format. 
                        Regards.

**Comment by nexopos** [comment]
Hi, It looks like your issue does'nt actually match the required format.
                        Please note that you can choose between the following format : 
                            - [Issue](https://github.com/nexopos/v3-issues/issues/new?template=bug_report.md).
                            - [Feature Submition](https://github.com/nexopos/v3-issues/issues/new?template=feature_request.md).
                        You might consider updating your issue and use the right version format. 
                        Regards.
